src/components/Savings/SavingsCard/index.tsx

```diff
@@ -112,9 +109,11 @@ export const SavingsCard = ({
                             </tr>
                         }
                     >
-                        {hasBalance && (
-                            <SavingsCardRow requiredChain={requiredChain} account={account} toggleModal={toggleModal} />
-                        )}
+                        <Web3SupportedNetworks
+                            onItem={({ chain }) => (
+                                <SavingsCardRow requiredChain={chain} account={account} toggleModal={toggleModal} />
```

Comment:
remember this one too... generally this PR looks like dup, total deja vu

Reply:
Some things might look the same as I had a merge with master..

But the actual fix is not a dupe
@roman only did the merging of tables for Stake page, I now did the same for portfolio page